docs_src/examples/Ch-ExamplesAddingMQTTDevice.md

```diff
@@ -250,16 +250,16 @@ Using the detailed script below as a simulator, there are three behaviors:
 
     **Using Multi-level Topic:**
     
-    1. The simulator receives the request from the MQTT broker, the topic is `command/my-custom-device/randnum` and the message is similar to the following:
+    1. The simulator receives the request from the MQTT broker, the topic is `command/my-custom-device/randnum/get/293d7a00-66e1-4374-ace0-07520103c95f` and message is similar to the following:
     
         ```
-        {"message":"42.0"}
+        {"randnum":"42.0"}
```

Comment:
do we need payload for the get command?

Reply:
no this is the message returned from the GET, clarified that in the documentation